keyboards/dlip/haritev2/thumbstick_pmw3389/rules.mk

```
@@ -0,0 +1,6 @@
+POINTING_DEVICE_ENABLE = yes
+SPI_DRIVER_REQUIRED = yes
+
+POINTING_DEVICE_DRIVER = pmw3389
+ANALOG_DRIVER_REQUIRED = yes
```

spi and analog driver are enabled by the joystick driver and pointing device driver config, as these devices require spi and analog, respectively.  So the specific driver enables aren't needed. 
```suggestion
POINTING_DEVICE_DRIVER = pmw3389
```